app/controllers/finance_overview_controller.rb

```diff
@@ -0,0 +1,23 @@
+class FinanceOverviewController < ApplicationController
+  before_action :authenticate_user!
+  after_action :verify_authorized
+
+  def index
+    authorize :finance_overview
+
+    @from_date = params[:from_date] || start_of_year
+    @to_date = params[:to_date] || end_of_year
+    @count_per_product = Order.count_per_product(@from_date, @to_date)
+    @count_per_category = Order.count_per_category(@from_date, @to_date)
+  end
+
+  private
+
+  def start_of_year
+    Time.zone.now.beginning_of_year.strftime('%Y-%m-%d')
```

Comment:
Maybe make this the start of a financial year (1 october) instead of a calendar year by default (same for end of year)?

Reply:
Main reason for this is that the discounts from grolsch for example are based on calendar year, and it is quite usefull for that I think. It also makes it easier code wise